docs/_sass/minima/_layout.scss

```diff
@@ -261,3 +261,12 @@
     width: calc(50% - (#{$spacing-unit} / 2));
   }
 }
+
+
+
```

Comment:
Believe it or not, the 3 line breaks here is for consistency with the rest of the file.